jedi/evaluate/docstrings.py

```diff
@@ -29,11 +29,13 @@
     r'\s*:type\s+%s:\s*([^\n]+)',  # Sphinx
     r'\s*:param\s+(\w+)\s+%s:[^\n]+',  # Sphinx param with type
     r'\s*@type\s+%s:\s*([^\n]+)',  # Epydoc
+    '\s*%s\s*\((.+?)\):.+',  # Google param with type
```

Comment:
Well, it's basically a context free language which would require to count the parenthesis, since that's not possible in a regular language it seemed like `.+?` would be the best solution here. We could try everything except for `:` instead but I'm not sure that will work correctly in all cases.

As for the `Args`, while not impossible, that would require a fairly large rewrite. The entire `_search_param_in_docstr` function wouldn't be usable anymore as it's only allowed to match using this regex when `Args:` precedes it. Or have I missed something in the code that would easily allow for this?